examples/ma2.py

```diff
@@ -24,12 +25,12 @@ def autocov(x, lag=1):
     """Autocovariance assuming a (weak) univariate stationary process with mean 0.
     Realizations are in rows.
     """
-    C = np.mean(x[:, lag:]*x[:, :-lag], axis=1, keepdims=True)
+    C = np.mean(x[:, lag:]*x[:, :-lag], axis=1)
     return C
 
 
 def discrepancy(x, y):
-    d = np.linalg.norm(np.array(x) - np.array(y), ord=2, axis=0)
+    d = np.linalg.norm(np.column_stack(x) - np.column_stack(y), ord=2, axis=1)
```

Comment:
I suppose this should be either with `np.row_stack` or `axis=0`, perhaps the previous is better to conform with the "row-per-sample" convention.